src/HelloServlet.java

```diff
@@ -0,0 +1,64 @@
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.util.*;
+
+@WebServlet("/HelloServlet")
+
+
+public class HelloServlet extends HttpServlet {
+
+
+    private String globalVar = null;
+
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        response.setContentType("text/html");
```

Comment:
Logic of sending response is duplicated several times. Better create helper method as you do in `printGlobalVarTest` 
Example:
```
private void sendResponse(String message, HttpServletRequest request, HttpServletResponse response)
{	
     response.setContentType("text/html");
     ...another code...
}
```